### PR TITLE
Update release info doc

### LINKED ISSUE #6997: Update Release Info for IDX Pages
**Overall change:**
Update [docs/Simorgh-Release-Info.md](https://github.com/bbc/simorgh/blob/latest/docs/Simorgh-Release-Info.md) with most read page details

**Code changes:**
- Add release dates for persian/afghanistan and ukrainian/ukraine_in_russian IDX pages and update Photogallery date
- Update scripts/simorghPages.js to include persian/afghanistan and ukrainian/ukraine_in_russian IDX pages
- run node scripts/simorghPages.js to update Release-info doc

### PR DESCRIPTION
**Pull request by thekp**
Resolves https://github.com/bbc/simorgh/issues/6997

**Overall change:**
_Updated simorgh release doc_

**Code changes:**

- _Added IDX page to releaseInfo script._
- _ran `node scripts/releaseInfo`_
- _Fixed a small linting issue._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: scripts/releaseInfo/index.js [hunk 1]
```diff
@@ -116,8 +122,13 @@ const generateLaunchDates = service => {
       output.push(`__On Demand Radio__: ${serviceLaunch.onDemandRadio}`);
     }
 
+    if (serviceLaunch.idxPage && serviceLaunch.idxPage !== '') {
+      output.push(`__IDX__: ${serviceLaunch.idxPage}`);
+    }
+
     return output.join('<br/>');
   }
+  return null;
```
Comment (thekp) [inline scripts/releaseInfo/index.js:131]: Prior to this change, eslint did not like that we had a `{}` curly bracket function that had no return

image

**Review by tochwill — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

**Comment by thekp** [comment]
image
How do people feel about this ^

We have ukrainianRu because our ukraine_in_russian page uses variant in the cypress config 👀 

I could update the release date to be on the ukrainianRu row?

**Comment by karinathomasbbc** [comment]
> I could update the release date to be on the ukrainianRu row?

That would work! Launch Dates is only for release info, so please amend as necessary. Thanks.

File: docs/Simorgh-Release-Info.md [hunk 2]
```diff
@@ -4,51 +4,51 @@ Remember to commit and push the changes to both launchDates.js and Simorgh-Relea
...
+| ukrainianRu | [IDX](http://localhost:7080/ukrainian/ukraine_in_russian) | [IDX](https://www.test.bbc.com/ukrainian/ukraine_in_russian) | [IDX](https://www.stage.bbc.com/ukrainian/ukraine_in_russian) | [IDX](https://www.bbc.com/ukrainian/ukraine_in_russian) |  |
```
Comment (amywalkerdev) [inline docs/Simorgh-Release-Info.md]: This puts the Ukrainian index in its own row as the service name is listed differently in our config..

Comment (amywalkerdev) [reply docs/Simorgh-Release-Info.md]: Oh I didn't refresh the page, you've already clocked it 😄

**Review by amywalkerdev — APPROVED** [review]
Looks good 👍

File: package-lock.json [hunk 3]
```diff
@@ -14201,7 +14201,7 @@
     },
     "jest-silent-reporter": {
       "version": "0.2.1",
-      "resolved": "https://registry.npmjs.org/jest-silent-reporter/-/jest-silent-reporter-0.2.1.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/jest-silent-reporter/-/jest-silent-reporter-0.2.1.tgz",
```
Comment (amywalkerdev) [inline package-lock.json]: Why have you done this? >:(

**Pull request merged**